lib/session/call-session.js

```diff
@@ -2069,9 +2069,7 @@ Duration=${duration} `
     }
     // When this call kicked out from conference, session need to replace endpoint
     // but this.ms might be undefined/null at this case.
-    if (!this.ms) {
-      this.ms = this.getMS();
-    }
+    this.ms = this.getMS();
```

Comment:
not what I asked for

Reply:
But

```
  /**
   * get a media server to use for this call
   */
  getMS() {
    if (!this.ms) {
      this.ms = this.srf.locals.getFreeswitch();
      if (!this.ms) {
        this._mediaServerFailure = true;
        throw new Error('no available freeswitch');
      }
    }
    return this.ms;
  }
```

so that this.ms = this.getMS(); is already enough

Reply:
I prefer the other because although it is functionally equivalent it signals clearly to the user that this is not needed in all cases, only in a special case

Reply:
I see, fixed